sdk/src/consts.ts

```diff
@@ -0,0 +1,2 @@
+export const DEFAULT_REF_TIME = 1250000000000
```

Comment:
Currently, there are two sources of constants: here and `.env`. We can eliminate all references to `.env`, such as `dotenv.config()`. Some constants are useful and should be transferred from `.env` to `consts.ts`. These constants include:

```bash
WAZERO_ADDRESS="5EFDb7mKbougLtr5dnwd5KDfZ3wK55JPGPLiryKq4uRMPR46"
ACCOUNT_URI="//Alice"
LOCAL="ws://localhost:9944"
CHAIN="alephzero-testnet"
```
Instead of loading from `.env` or using hardcoded values, we can import them from `consts.ts`